pysmt/test/test_lira.py

```diff
@@ -46,6 +47,36 @@ def test_toreal(self):
         self.assertEqual(ToReal(Plus(b, Int(1))),
                           ToReal(ToReal(Plus(b, Int(1)))))
 
+    def test_realtoint(self):
+        b = Symbol('b', INT)
+
+        check = Equals(RealToInt(Real(Fraction(3, 2))), Int(1))
+        check = And(check,
+                    Equals(RealToInt(Real(Fraction(-3, 2))), Int(-2)))
+        check = And(check,
+                    Equals(RealToInt(Real(1)), Int(1)))
+        check = And(check,
+                    Equals(RealToInt(ToReal(b)), b))
+        for sname in get_env().factory.all_solvers(logic=UFLIRA):
```

Comment:
the logic here should be 'QF_LIRA' to enable all the relevant solvers.

Reply:
It seems `QF_LIRA` is not defined in `pysmt.logics`. Should I define it?